### DISCUSSION TITLE
Search is not working!

### DISCUSSION THREAD

**Discussion by YuchanJeong** [jpanther/congo#153] [Q&A] [answered]
```bash
enableSearch = true
```

The search feature works only locally(hugo server). In the deployed environment, the search box does not appear when I click search icon.

github: https://github.com/YuchanJeong/_blog
blog: https://yuchanjeong.github.io/

**Comment by wizardbyron** [comment, marked as answer]
I checked your site that there is an error message if you open your browser debug console:

```
Cannot load script https://yuchanjeong.github.io/js/main.bundle.min.25aee3c19fd055045705d18e76ae055c4557be64be3f17819df0f61f8356ea5014259eb52e72e4cf3934d72ea0a40cd17afd50f7ecd8ec1db3cbc74a04aa0756.js. Failed integrity metadata check. Content length: 28160, Expected content length: -1, Expected metadata: sha512-Ja7jwZ/QVQRXBdGOdq4FXEVXvmS+PxeBnfD2H4NW6lAUJZ61LnLkzzk01y6gpAzRev1Q9+zY7B2zy8dKBKoHVg==
```

**Reply by YuchanJeong** [reply]
Thank you! I will check and if I find something I'll upload :)

**Comment by rishikeshsreehari** [comment]
Hi I'm not able to resolve the search issue. @YuchanJeong  How did you fix it?

**Reply by Yaostars** [reply]
Hello, I also have such problem, how you solve it? Can you tell me some steps?

**Comment by Yaostars** [comment]
Hello, how did you solve the problem? I am a fresh man for website construction. Can you teach me some methods?
